src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -1070,20 +1088,49 @@ private void StartDatabaseOnTimer(string name, DateTime? wakeup = null)
-                          {
-                              var ex = t.Exception.ExtractSingleInnerException();
-                              if (ex is DatabaseConcurrentLoadTimeoutException e)
-                              {
-                                  // database failed to load, retry after 1 min
-                                  ForTestingPurposes?.RescheduleDatabaseWakeupMre?.Set();
+                    switch (nextIdleDatabaseActivity.Type)
+                    {
+                        case IdleDatabaseActivityType.UpdateBackupStatusOnly:
+                            var backupStatus = BackupUtils.GetBackupStatusFromCluster(_serverStore, databaseName, nextIdleDatabaseActivity.TaskId);
+                            
+                            backupStatus.LastIncrementalBackup = backupStatus.LastIncrementalBackupInternal = nextIdleDatabaseActivity.DateTime;
+                            backupStatus.LocalBackup.LastIncrementalBackup = nextIdleDatabaseActivity.DateTime;
+                            backupStatus.LocalBackup.IncrementalBackupDurationInMs = 0;
+
+                            var backupResult = new BackupResult();
+                            backupResult.AddMessage($"Skipping incremental backup because no changes were made from last full backup on {backupStatus.LastFullBackup}.");
+
+                            BackupUtils.SaveBackupStatus(backupStatus, databaseName, _serverStore, _logger, backupResult);
+                            
+                            nextIdleDatabaseActivity = BackupUtils.GetEarliestIdleDatabaseActivity(new BackupUtils.EarliestIdleDatabaseActivityParameters
+                            {
+                                DatabaseName = databaseName,
+                                LastEtag = nextIdleDatabaseActivity.LastEtag,
+                                Logger = _logger,
+                                ServerStore = _serverStore
+                            });
 
-                                  if (_logger.IsInfoEnabled)
-                                      _logger.Info($"Failed to start database '{name}' on timer, will retry the wakeup in '{_dueTimeOnRetry}' ms", e);
+                            RescheduleNextIdleDatabaseActivity(databaseName, nextIdleDatabaseActivity);
+                            break;
 
-                                  RescheduleDatabaseWakeup(name, milliseconds: _dueTimeOnRetry, wakeup);
-                              }
-                          }, TaskContinuationOptions.OnlyOnFaulted);
+                        case IdleDatabaseActivityType.WakeUpDatabase:
+                            _ = TryGetOrCreateResourceStore(databaseName, nextIdleDatabaseActivity.DateTime).ContinueWith(t =>
+                            {
+                                var ex = t.Exception.ExtractSingleInnerException();
+                                if (ex is DatabaseConcurrentLoadTimeoutException e)
+                                {
```

Comment:
Why do we need to handle this case?
The database will be loaded anyway, no?

Reply:
The decision to handle this specific exception type was made based on the following issue: https://issues.hibernatingrhinos.com/issue/RavenDB-18420/Database-might-not-wake-up-from-idle-if-we-are-waking-a-lot-of-databases-from-idle-state